src/ResourceManager/AzureBatch/Commands.Batch/Locations/GetBatchLocationQuotasCommand.cs

```diff
@@ -18,8 +18,8 @@
 
 namespace Microsoft.Azure.Commands.Batch
 {
-    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchSubscriptionQuotas), OutputType(typeof(PSBatchSubscriptionQuotas))]
-    public class GetBatchSubscriptionQuotasCommand : BatchCmdletBase
+    [Cmdlet(VerbsCommon.Get, Constants.AzureRmBatchLocationQuotas), OutputType(typeof(PSBatchLocationQuotas))]
```

Comment:
@markcowl, thanks for the info.  I apologize though, because I'm still kind of fuzzy on what exactly you're asking me to do.  Can you elaborate a bit on what the ask is here?
The output type has changed names, but no properties or parameter sets have been altered.  Scripts that use the cmdlet (with the alias added) will not be broken.  This will still work:
$quotas = Get-AzureRmBatchSubscriptionQuotas "westus";
Write-Host "My account quota in westus is $quotas.AccountQuota";

There's no cmdlet that accepts this object as an input parameter, so users won't be instantiating their own.  Quota changes are currently handled via support ticket only, so this API is only about giving users info on what their current quota value is.  There's no quota-related APIs that would use this object.

Feel free to ping me if it would be easier.  Thanks.